### PR TITLE
Add names to right in names-generator

### PR DESCRIPTION
**Pull request by lonewarrior556**
Signed-off-by: Konrad <lonewarrior556@yahoo.com>
**- What I did**
Added a few names seriosu and a few fun names to name-generator to slightly increase variation.

**- How I did it**
Following current convention

**- How to verify it**
see below

**- Description for the changelog**
Add a few names to names-generator to slightly increase variation.

image

### CODE REVIEW

**Comment by coolljt0725** [comment]
ping @thaJeztah

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -260,6 +269,9 @@ var (
 		// Leonhard Euler invented large parts of modern mathematics. https://de.wikipedia.org/wiki/Leonhard_Euler
 		"euler",
 
+		// Hubert J. Farnsworth proprietor of the Planet Express delivery service - https://en.wikipedia.org/wiki/Professor_Farnsworth
+		"farnsworth",
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Same for farnsworth :smile:

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -452,6 +467,9 @@ var (
 		// Ian Murdock - founder of the Debian project - https://en.wikipedia.org/wiki/Ian_Murdock
 		"murdock",
 
+		//Randall Munroe  - an American cartoonist, author, engineer, scientific theorist, and the creator of the XKCD phone https://en.wikipedia.org/wiki/Randall_Munroe
+		"munroe",
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Big fan, but not sure it fits the general theme 😅

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thank you for contributing; I left some comments inline, but we also try to have an equal representation of female and male people in the list; could you update the PR, addressing my comments, and adding some women to keep it balanced?

**Comment by lonewarrior556** [comment]
:) I can add distinguished females, however is it possible if at least one of the trio (`brannigan`, `farnsworth`, `Munroe`) made it? (`farnsworth` would be my choice) The sole reason they were added was because I saw `Emmett Brown` is in there!

**Comment by thaJeztah** [comment]
:smile: I know it's tempting; there's two jokes in the file; "Emmet Brown" (as you found), and of course [the Woz](https://github.com/moby/moby/blob/4f0d95fa6ee7f865597c03b9e63702cdcb0f7067/pkg/namesgenerator/names-generator.go#L635-L639); not sure if we should add more, or keep it to "it's funny once".

Perhaps there's something to say for both of those being male (so should we add [Doctor Amy Wong](https://en.wikipedia.org/wiki/Amy_Wong)?)

**Comment by AkihiroSuda** [comment]
ping @lonewarrior556

also please sign your commit with your full name

**Comment by cpuguy83** [comment]
I'm going to close this due to inactivity.
Please let me know if you are still interested in submitting these and we can re-open.

Thanks! 🙇

**Pull request closed without merging**